### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by praspatil05**
Making changes for new TPM versions.

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -562,6 +562,14 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 A recent fix has addressed an issue where some Guest users were being omitted from the output of the Get-CsOnlineUser cmdlet, resulting in an increase in the reported user count.
 
+**Updates in Teams PowerShell Module version 6.1.0 and later**
+
+The following updates are applicable for organizations having TeamsOnly users that use Microsoft Teams PowerShell version 6.1.0 or later.
```
Comment (shivtorov) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:567]: very confusing line.
"organizations having TeamsOnly users that use Microsoft Teams PowerShell version 6.1.0"

- are you referring to tenant-level setting (`Get-CsTenant | select TeamsUpgradeEffectiveMode`)?

Comment (NubeNueve) [reply skype/skype-ps/skype/Get-CsOnlineUser.md:567]: @Milehigh-world

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ This parameter is added to Get-CsOnlineUser starting from TPM 4.5.1 to indicate
 - `ResourceAccount` - to query for app endpoints or resource accounts.
 - `Guest` - to query for guest accounts.
 - `SfBOnPremUser` - to query for users that are hosted on-premises (available from January 31, 2023, in the latest TPM versions at that time).
-- `Unknown` - to query for a user type that is not known.
+- `Unknown` - to query for a user type that is not known. (This AccountType is being renamed to IneligibleUser)
```
Comment (NubeNueve) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:130]: No

**Review by NubeNueve — COMMENTED** [review]
Evanmichael/milehigh.world

**Pull request closed without merging**